.travis.yml

```diff
@@ -3,7 +3,10 @@ language: node_js
 node_js:
   - "6"
 
-sudo: false
```

Comment:
hmm, I think we should probably keep this line to make sure we're using containers 🤔